### PR TITLE
Feature/bookmark emotion entry

### PR DESCRIPTION
**Pull request by Leon910**

### CODE REVIEW

File: components/BookmarkButton/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import BookmarkPinIcon from "assets/bookmarkicons/BookmarkPinIcon.svg";
+
+export default function BookmarkButton({ isBookmarked, onToggleBookmark }) {
+  return (
+    <>
+      <StyledButton onClick={onToggleBookmark} aria-label="Toggle Bookmark">
+        <StyledBookmarkPinIcon $isBookmarked={isBookmarked} />
+      </StyledButton>
+    </>
```
Comment (onemanwenttomow) [inline components/BookmarkButton/index.js]: If we are only returning one element from the component, then we don't need the react fragment. 

```suggestion
      <StyledButton onClick={onToggleBookmark} aria-label="Toggle Bookmark">
        <StyledBookmarkPinIcon $isBookmarked={isBookmarked} />
      </StyledButton>
```

File: components/EmotionDetails.js [hunk 2]
```diff
@@ -4,7 +4,12 @@ import styled from "styled-components";
...
 import { useState } from "react";
 
-export default function EmotionDetails({ emotion, onDeleteEmotion }) {
+export default function EmotionDetails({
+  emotion,
+  onDeleteEmotion,
+  myBookmarkedEmotions,
+  onToggleBookmark,
+}) {
   const [isDialogOpen, setIsDialogOpen] = useState(false);
```
Comment (onemanwenttomow) [inline components/EmotionDetails.js:13]: You could consider looking into the [`<dialog>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) element.

File: components/Navigation/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { useRouter } from "next/router";
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function Navigation() {
+  const router = useRouter();
```
Comment (onemanwenttomow) [inline components/Navigation/Navigation.js]: I think we can remove this as we are not using in this component.

File: components/Navigation/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { useRouter } from "next/router";
```
Comment (onemanwenttomow) [inline components/Navigation/Navigation.js]: I think we can remove this as we are not using in this component.

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Hey team, nice work! 

I just have a couple of things that should be changed, and one suggestion (to look into the dialog html element). No need to implement the `dialog` element though

File: components/Navigation/Navigation.js (hunk 4 above)
Comment (Leon910) [reply components/Navigation/Navigation.js]: Hi Pete! Thanks for your great hints! I removed the unnecessary fragments as well as the router hook. The `dialog`-element we might consider to implement in one of our next US.

**Review by onemanwenttomow — APPROVED** [review]
LGTM 🎉

**Pull request merged**